archesproject/arches
# 8076 fix related resource broken config

to test this create 2 models (A and B)
Model A has a resource instance node that references Model B
Then delete Model B
Go back to Model A and select the resource instance node.  Notice that the UI is broken and can not be recovered from.
Apply the fix and notice that you can now reassign the graph reference.
